addons/mail/static/src/js/chatter.js

```diff
@@ -314,6 +320,11 @@ var Chatter = form_common.AbstractField.extend({
             this.followers.on('followers_update', this, this.on_followers_update);
         }
 
+        if(!this.options.no_activity) {
+            self.activity = new ActivityLog(self.view.datarecord.id, self.view.model);
+            self.activity.replace(self.$('.o_mail_activity'));
```

Comment:
we have `o_mail_activity`  as activity holder div in chatter so instead of replace I append activity to `o_mail_activity`